samcli/lib/sync/flows/alias_version_sync_flow.py

```diff
@@ -62,19 +63,33 @@ def __init__(
         self._alias_name = alias_name
         self._lambda_client = None
 
+    @property
+    def sync_state_identifier(self) -> str:
+        """
+        Sync state is the unique identifier for each sync flow
+        In sync state toml file we will store
+        Key as AliasVersionSyncFlow:FunctionLogicalId:AliasName
+        Value as alias version number
+        """
+        return self.__class__.__name__ + ":" + self._function_identifier + ":" + self._alias_name
+
     def set_up(self) -> None:
         super().set_up()
         self._lambda_client = self._boto_client("lambda")
 
     def gather_resources(self) -> None:
         pass
 
+    def compare_local(self) -> bool:
```

Comment:
SyncFlow class already has its own compare_local method, to me since for alias version sync flow they will always have no local sha value during comparison, we can just override the method to return False. But removing this method will have the same effect, so I'm fine with removal

Reply:
Ah OK it makes sense now, thanks for explaining!